splinterd/src/node/runnable/admin.rs

```diff
@@ -171,6 +177,25 @@ impl RunnableAdminSubsystem {
                 }
             };
 
+        let mut attempts = 0..MAX_STARTUP_WAIT_ATTEMPTS;
+
+        loop {
+            let status = commands
+                .admin_service_status()
+                .map_err(|e| InternalError::from_source(Box::new(e)))?;
+
+            if status == AdminServiceStatus::Running {
+                break;
+            }
+            if attempts.next().is_none() {
+                return Err(InternalError::with_message(format!(
+                    "Admin service has completely started after {} secs",
```

Comment:
Is the intended message here that the admin service hasn't started?

Reply:
Fixed